
Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-j6mwr-4[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-vtp68[0m
{"level":"warn","ts":1592328813.3640854,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1592328813.3650248,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-j6mwr-4[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-lwlwb[0m
{"level":"warn","ts":1592328815.381393,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1592328815.3844323,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-j6mwr-4[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-j6mwr-4[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-seldonio-seldon-core-pr-19-rv942[0m
{"level":"warn","ts":1592328821.2444308,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1592328825.7860053,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ e14f94ad582672c53ea11c97c8ddcce0a751ac47 in path /workspace/source"}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-j6mwr-4[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:e14f94ad582672c53ea11c97c8ddcce0a751ac47,1952:65dfd7e6e13f983dd5b19b9f721826fcd54ed3b3
DEBUG: ran git fetch --unshallow origin 65dfd7e6e13f983dd5b19b9f721826fcd54ed3b3: e14f94ad582672c53ea11c97c8ddcce0a751ac47: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard e14f94ad582672c53ea11c97c8ddcce0a751ac47 in 
DEBUG: ran clean --force -d . in 
error: error during merge: merging 65dfd7e6e13f983dd5b19b9f721826fcd54ed3b3 into master: git output: Auto-merging executor/go.mod
CONFLICT (content): Merge conflict in executor/go.mod
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge 65dfd7e6e13f983dd5b19b9f721826fcd54ed3b3' command in directory '', output: 'Auto-merging executor/go.mod
CONFLICT (content): Merge conflict in executor/go.mod
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m
